source/common/stats/stats_impl.cc

```diff
@@ -68,14 +68,18 @@ TagExtractorImpl::TagExtractorImpl(const std::string& name, const std::string& r
     : name_(name), prefix_(std::string(extractRegexPrefix(regex))),
       regex_(RegexUtil::parseRegex(regex)) {}
 
+static bool regexStartsWithDot(absl::string_view regex) {
```

Comment:
Nit: can you move this to anonymous namespace to match general Envoy style for `.cc` local definitions?

Reply:
I made that exact comment on @gsagula 's gzip filter review and dpn commented on my comment:

"I think in most cases of Envoy code base you will see static methods rather than the anonymous namespaces style (I told @gsagula to use the static method style based on that), although there's plenty of anonymous namespace style too. The google style guide is ambivalent about this. Let's standardize on one or the other and codify it in STYLE.md."

Shame on me for not following up.  On this particular topic I actually truly don't care -- static vs anon-namespace.  Can the two of you decide and I'll write the PR to put it in STYLE.md and (if needed) fix this instance accordingly?

Reply:
I vote for anonymous namespaces as the standard.